### PR TITLE
DCT-317

### PR DESCRIPTION
**Pull request by EBEREGIT**
Error Message 119

Added Code Sample

### CODE REVIEW

File: docs/src/rsh/errors/index.md [hunk 1]
```diff
@@ -1371,10 +1371,10 @@ there needs to be a timeout associated with the publication if `{!rsh} shouldPub
 
 In `{!rsh} parallelReduce` or `{!rsh} fork`, a `{!rsh} timeout` is required unless one `{!rsh} Participant`
 always races, the `{!rsh} when` field in their `PUBLISH_EXPR` is statically `{!rsh} true`, or if one
-`{!rsh} ParticipantClass` will attempt to race.
+`{!rsh} ParticipantClass` or `{!rsh} API` will attempt to race.
 
 :::note
-Since `ParticipantClass` is being deprecated, it will be preferable to use `API` in place of `ParticipantClass`.
```
Comment (TheChronicMonster) [inline docs/src/rsh/errors/index.md]: The monospaced copy should be `{!rsh} ParticipantClass` and `{!rsh} API`

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

File: docs/src/rsh/errors/index.md [hunk 2]
```diff
@@ -2116,6 +2116,12 @@ Maybe(UInt).Some(5).match({
 This error indicates that you have inspected the details about a publication, such as via `{!rsh} didPublish()`, before there has been a publication.
 This is impossible, so the expression must be moved after the first publication.
 
+```reach
+  didPublish(handAlice)
+  Alice.publish(handAlice);
+  commit();
+```
+
```
Comment (TheChronicMonster) [inline docs/src/rsh/errors/index.md:2124]: This entry is incomplete. 
We need to explicitly state that this sample is erroneous and then create a sample that is correct and explicitly state that the example is the proper way to use `didPublish()`

File: docs/src/rsh/errors/index.md [hunk 3]
```diff
@@ -2116,6 +2116,12 @@ Maybe(UInt).Some(5).match({
 This error indicates that you have inspected the details about a publication, such as via `{!rsh} didPublish()`, before there has been a publication.
 This is impossible, so the expression must be moved after the first publication.
 
+```reach
+  didPublish(handAlice)
```
Comment (TheChronicMonster) [inline docs/src/rsh/errors/index.md:2120]: Missing a semicolon.

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

**Comment by jeapostrophe** [comment]
Please request a review from me once @TheChronicMonster approves

**Comment by jeapostrophe** [comment]
This commit can no longer be merged because you did a merge rather than a rebase

**Pull request closed without merging**